schemas/error-schema.json

```diff
@@ -60,6 +60,12 @@
               "description": "The type of the error",
               "readOnly": true
             },
+            "handling": {
+              "type": "string",
+              "description": "Whether the error has been handled manually in the source code or not",
+              "enum": ["caught", "uncaught"],
+              "readOnly": true
+            },
```

Comment:
I'd like to discuss the name/value of this property a bit more.

* I think "caught" relates to a `try/catch` usage. But:
  * should a `console.error` be marked as "caught", since it has no `catch` involved?
  * rejected promises are generally associated with `handled/unhandled` instead of `caught/uncaught`

* Since "caught" and "uncaught" values are very binary (anything can be either caught or uncaught), I don't think we cannot add more value to this enum. So, is it useful to have an enum instead of a boolean property?

In the RFC, we proposed `is_handled: bool`, and I wonder why we chose to implement it differently. I think the initial proposal is better.

Reply:
I totally agree on the fact this could be a boolean instead of an enum. Also the handled/unhandled does seem more relevant than caught/uncaught as it applies to more use case than just a try-catch situation.

Reply:
I also agree on these points. To me `is_handled` was more straightforward and less confusing for the reasons already explained above.

Reply:
Hello 👋, those are valid points 😬

In this case, would it be relevant to have a facet `Error Handling: handled/unhandled`? I do not really like boolean facets and the fact that we should have a long statement in the facet name like `Error Is Handled`, also it forces us to promote a value (in this case handled errors are promoted)

If you all disagree with ^ I am fine to keep `Error Is Handled: true/false` 👍

Reply:
Having a enum rather than a boolean would indeed be more explicit in the UI.

Reply:
We already have multiple boolean fields: `is_crash`, `is_active`, `in_foreground`, `has_replay`. Using `is_handled` would be more in line with what we already have. If it's an UI issue, we may imagine changing the UI?

Reply:
for the other boolean fields, it seems that they are either displayed as an icon or not being that important in the UI.
IMO, a solution avoiding to display true/false would be great.
I don't mind not using a boolean in this case if it makes things clearer for customers.

Reply:
If that make more sense to the end user, it seems reasonable :)